README.rst

```diff
@@ -1,20 +1,22 @@
 ======================
-Plone Example Buildout
+Plone Simple Buildout
 ======================
 
 A basic, cross-platform `Plone <https://plone.org>`_ buildout.
 
+This installation method is simpler than the `Plone unified installer <https://github.com/plone/Installers-UnifiedInstaller>`_ for UNIX-based platforms, which automates the process of installing required software and setting up a virtual environment.
+
```

Comment:
This confuses at least me :)
If you are new and on UNIX, IMHO UI is better as in easier.